website/src/pages/downloads.tsx

```diff
@@ -103,18 +104,76 @@ const VersionSelect = () => {
     );
 };
 
-const Asset = ({ url }) => {
+const Asset = ({ urls }) => {
+    let asset: string;
+    let gpgSig: string;
+    let coCert: string;
+    let coSig: string;
+    let name: string = "Binary";
+
+    for (let url of urls) {
+        if (url.toLowerCase().includes(".deb")) {
+            name = "Debian Package";
+        } else if (url.toLowerCase().includes(".rpm")) {
+            name = "RPM Package";
+        } else if (url.toLowerCase().includes(".pkg") && url.toLowerCase().includes("linux")) {
+            name = "Alpine Package";
+        }
+
+        if (url.toLowerCase().includes(".gpgsig")) {
+            gpgSig = url;
+            continue;
+        }
+        if (url.toLowerCase().includes(".pem")) {
+            coCert = url;
+            continue;
+        }
+        if (url.toLowerCase().includes(".sig")) {
+            coSig = url;
+            continue;
+        }
+        asset = url;
+    }
+
+    if (asset === undefined) {
+        console.trace("UNKOWN ASSET: ", asset, urls, gpgSig, coCert, coSig);
+        return;
+    }
+
     const { selectedItem } = useOptions();
     return (
         <div className="pagination-nav__item">
-            <a className="pagination-nav__link" href={url}>
-                <div className="pagination-nav__label">
-                    {AssetArchitecture(url).toUpperCase()}
+            <div className="card">
```

Comment:
Can you please give the cards a stronger boarder? They are very difficult to keep apart, especially in dark mode.
**New**:
![image](https://github.com/user-attachments/assets/45c3f7e8-5278-4676-af80-f240a5880c7c)

**Old**:
![image](https://github.com/user-attachments/assets/a1ef75bf-0353-4bab-a484-00e8edb2fb6d)

Reply:
Interesting, yeah, I see the borders aren't good. I suspect it is because it is a nested card (the outer border is also a card) so perhaps it truncated one of the borders as a result, I'm not sure. I'll play with it!

Reply:
![image](https://github.com/user-attachments/assets/ba0a00ee-d1fb-4edd-9c4a-f1a94848b913)

I think I fixed this by changing the background color and fixing the "light" shadow color.